jira-bot.js

```diff
@@ -61,7 +61,7 @@ class JiraBot extends EventEmitter {
       this.slack.sendMessage(`@${user.name} hello :)`, message.channel);
     }
 
-    issueKeys = text.match(this.issueKeysRegex) || [];
+    issueKeys = text.match(this.issueKeysRegex).reverse() || [];
```

Comment:
This change won't fix messages order. `match` returns matches in in direct order, but ticket info posted as soon as it found in Jira. Therefore if the bot find two issue keys in the Slack message but Jira responds faster to the second request, tickets info will be posted in reverse order.

PS: you shouldn't try to reverse an array of matches just by adding `.reverse` after `match()`. `match()` returns `null` if there are no matches and you'll get a `TypeError` in that case.